ebusd-2.1.x/de/vaillant/08.recov.csv

```diff
@@ -0,0 +1,38 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
+#,recov,recoVair 260,,,,,,,,,,,
+# ##### Generell #####,,,,,,,,,,,,,
+*r,,,,,,B513,,,,IGN:2,,,
+*w,,,,,,B509,,,,,,,
+r,recov,VentIntensityDay,Lueftungsintensitaet Tag auslesen,,,,048a03,,,UCH,,,
```

Comment:
the circuit can be left out when either the file is intended fro use with scanconfig or the default lines contain the circuit

Reply:
@john30  I don't understand the comment. I'm too dumb for that...